wormhole-connect/src/components/ConnectWallet.tsx

```diff
@@ -99,9 +99,7 @@ function ConnectWallet(props: Props) {
     (popupState?: any) => {
       if (disabled) return;
       if (popupState) popupState.close();
-      if (side === 'destination' && manualAddressTarget) {
```

Comment:
Have you checked if the close without connection works after this change?

Reply:
if you mean that when you change wallet and just close it, the current state is maintained, yes.